### DISCUSSION TITLE
Manual/SL-BuiltinFunctions

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#669] [Notes]
# Manual/SL-BuiltinFunctions

https://docs.unity3d.com/Manual/SL-BuiltinFunctions.html

**Comment by SoylentGraham** [comment]
// The function I always need, but isn't built in
float3 UnityObjectToWorldPos(float3 LocalPos)
{
	return mul( unity_ObjectToWorld, float4( LocalPos, 1 ) ).xyz;
}
